### PR TITLE
sync via modulesync

### PR DESCRIPTION
**Pull request by mhaskel**

### CODE REVIEW

**Comment by daenney** [comment]
:-1: 

Please don't remove all the strict vars and other settings.

File: Gemfile [hunk 1]
```diff
@@ -11,23 +11,18 @@ def location_for(place, fake_version = nil)
 end
 
 group :development, :unit_tests do
-  gem 'rake',                    :require => false
   gem 'rspec-core', '3.1.7',     :require => false
-  gem 'rspec-puppet', '~> 2.1',  :require => false
```
Comment (daenney) [inline Gemfile]: Please don't remove this line, we need at least 2.1 for Puppet 4 support

Comment (mhaskel) [inline Gemfile]: I can add this line back

**Comment by mhaskel** [comment]
@daenney We need to severely trim the matrix, the modules pipelines are making travis-ci unusable across the organization. So, we were figuring strict/future parser would be consistent enough across puppet 3 and puppet 4.

**Comment by daenney** [comment]
Sure, trim the matrix but keep `STRICT_VARIABLES=yes` everywhere please. I don't want to get into fights with PRs that pass or not on 3 vs. 4 over inconsistent strict vars.

**Comment by mhaskel** [comment]
ok.

**Comment by mhaskel** [comment]
I'm going to close and resubmit, cause modulesync confusion made things get weird.

**Pull request closed without merging**